.gitignore

```diff
@@ -78,6 +78,7 @@ npm-debug.log.*
 !/upload/.htaccess
 
 /docs/phpdoc-sf/
+/docs/hooks/
```

Comment:
See comment about default option value, but I don't think we should use this folder as default value The parameter should be required